testing/testcafe/tests/dataGrid/columnChooser.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import DataGrid, { CLASS } from '../../model/dataGrid';
+import { getData } from './helpers/generateDataSourceData';
+
+fixture.disablePageReloads`Column chooser`
+  .page(url(__dirname, '../container.html'));
+
+test('Column chooser screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getColumnChooserButton())
+    // act
+    .expect(await takeScreenshot('column-chooser', dataGrid.element))
```

Comment:
It's better to add the extension to the screenshot name:

```suggestion
    .expect(await takeScreenshot('column-chooser.png', dataGrid.element))
```